Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+    agent any
+
+    stages {
+
+        stage("Clean") {
+            steps {
+                sh "./gradlew clean"
+            }
+        }
+
+        stage("Assemble") {
+            steps {
+                sh "./gradlew assemble --refresh-dependencies -i"
```

Comment:
`--refresh-dependencies` makes it VERY slow and is used to solve issues, not intended for a standard build